8 0 beta (#10592)
qooxdoo/qooxdoo: docs/core/understanding_properties.md

- `isPropertyName()`: Identical to `getPropertyName()`.
- `togglePropertyName()`: Toggles between true and false.

## First-class nature of properties
Properties are first-class members of a class. They can be manipulated
as if they were declared in the `member` section of the configuration
-- they can be assigned to, or their value read as if it were a member
variable -- but they are treated as if the functional setter or

> getter, were called. All operations desscribed below, that would occur

Reviewer: ```suggestion
getter, were called. All operations described below, that would occur
```

as a result of calling the functional setter, are also called by
assigning to the property as a variable.

By way of example, for property `myProp`, these two getter statements are equivalent:

> 

Reviewer 2: Same with these code snippet, add js lang

Author: fixed

There are two ways to set an init value of a property.

#### Init value, calculated at class instantiation time

> The _preferred_ way for reference vaues such as objects or arrays,

Reviewer: ```suggestion
The _preferred_ way to initialize reference vaues such as objects or arrays,
```

which are not expected to be shared between instances of the class, is
to provide a function to the `initFunction` key in the property
configuration map. The function will be called each time the class is
instantiated, and should return the initial value for the property in
that instance. You can use this key standalone or in combination with
`nullable` and/or `inheritable`.

```javascript
properties : {
  myProperty : {

>     initFunction : function() { return [ 1, 2, 4, 8 ]; }

Reviewer: ```suggestion
    initFunction : () => { return [ 1, 2, 4, 8 ]; }
```
